rskj-core/src/test/java/co/rsk/peg/federation/FederationChangeIT.java

```diff
@@ -679,6 +720,18 @@ private Pair<BtcTransaction, Keccak256> createFlyoverPegin(Federation federation
         return Pair.of(flyoverPeginBtcTx, flyoverDerivationHash);
     }
 
+    private BtcTransaction createPegout(Federation federation, String senderSeed) {
+        var pegout = new BtcTransaction(NETWORK_PARAMS);
+        pegout.addInput(BitcoinTestUtils.createHash(1), 0, createBaseP2SHInputScriptThatSpendsFromRedeemScript(federation.getRedeemScript()));
+
+        var receiverPublicKey = BitcoinTestUtils.getBtcEcKeyFromSeed(senderSeed);
+        pegout.addOutput(Coin.COIN, receiverPublicKey);
+        var federationP2SHScript = federation.getP2SHScript();
+        pegout.addOutput(Coin.COIN.multiply(10), federationP2SHScript);
```

Comment:
Don't you think this value is too big? What about using something more realistic close to the minimum pegin or pegout?